### PR TITLE
`dc` to newshell

### PR DESCRIPTION
**Pull request by MalhotraPulak**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [x] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

A direct port of the `dc` commands from the old shell to new shell, did not alter logic of any command. As always open to feedback! I tested them in a debug session and compared them to old shell, seemed to work exact same for me. 
...

**Test plan**

CI is green

...

**Closing issues**

...

### CODE REVIEW

File: librz/core/agraph.c [hunk 1]
```diff
@@ -4027,7 +4027,7 @@ static void graph_breakpoint(RzCore *core) {
 }
 
 static void graph_continue(RzCore *core) {
-	rz_debug_continue_oldhandler(core, "");
+	rz_cmd_debug_continue_execution_handler(core, 1, NULL);
```
Comment (XVilka) [inline librz/core/agraph.c]: What about using the proper API instead?
```c
CMD_CHECK_DEBUG_DEAD(core);
rz_cons_break_push(static_debug_stop, core->dbg);
rz_reg_arena_swap(core->dbg->reg, true);
#if __linux__
core->dbg->continue_all_threads = true;
#endif
rz_debug_continue(core->dbg);
dbg_follow_seek_register(core);
```
Or making it some function in the `librz/core/cdebug.c` and using it instead?
Reusing command handlers for pure API purpose doesn't seem right.

File: librz/core/panels.c [hunk 2]
```diff
@@ -3345,7 +3345,7 @@ int __hexpairs_cb(void *user) {
 
 int __continue_cb(void *user) {
 	RzCore *core = (RzCore *)user;
-	rz_debug_continue_oldhandler(core, "");
+	rz_cmd_debug_continue_exception_handler(core, 1, NULL);
```
Comment (XVilka) [inline librz/core/panels.c]: Same here.

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/core/cmd_debug.c [hunk 3]
```diff
@@ -4781,9 +4611,9 @@ RZ_IPI int rz_cmd_debug(void *data, const char *input) {
 	case 'H': // "dH"
 		eprintf("TODO: transplant process\n");
 		break;
-	case 'c': // "dc"
-		(void)rz_debug_continue_oldhandler(core, input + 1);
-		break;
+		//	case 'c': // "dc"
```
Comment (XVilka) [inline librz/core/cmd_debug.c]: Just remove this

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/core/core_private.h [hunk 4]
```diff
@@ -127,4 +127,8 @@ RZ_IPI void rz_core_meta_comment_add(RzCore *core, const char *comment, ut64 add
 
 /* cmd_flag.c */
 RZ_IPI void rz_core_flag_describe(RzCore *core, ut64 addr, bool strict_offset, RzOutputMode mode);
+
+/*cmd_debug.c */
```
Comment (XVilka) [inline librz/core/core_private.h]: Missing space here.

File: librz/core/cdebug.c [hunk 5]
```diff
@@ -72,7 +72,13 @@ RZ_API bool rz_core_debug_step_one(RzCore *core, int times) {
...
 RZ_IPI void rz_core_debug_continue(RzCore *core) {
 	if (rz_config_get_b(core->config, "cfg.debug")) {
-		rz_debug_continue_oldhandler(core, "");
+		rz_cons_break_push(rz_core_static_debug_stop, core->dbg);
+		rz_reg_arena_swap(core->dbg->reg, true);
+#if __linux__
+		core->dbg->continue_all_threads = true;
+#endif
+		rz_debug_continue(core->dbg);
+		rz_core_dbg_follow_seek_register(core);
```
Comment (XVilka) [inline librz/core/cdebug.c:82]: Missing `rz_cons_break_pop()`

**Review by XVilka — CHANGES_REQUESTED** [review]
Apart from that comment, everything else looks ok.

**Review by wargio — APPROVED** [review]

**Review by XVilka — APPROVED** [review]

**Comment by XVilka** [comment]
Tested and can confirm as well.

**Pull request merged**